test/controllers/folio/console/api/ai_assistant_controller_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class Folio::Console::Api::AiAssistantControllerTest < Folio::Console::BaseControllerTest
+  include Devise::Test::IntegrationHelpers
+  include Folio::Engine.routes.url_helpers
+
+  def setup
+    create_and_host_site
+    @admin = create(:folio_account)
+    sign_in @admin
+  end
+
+  test "invalid" do
+    record_without_assistant = create(:folio_page)
+
+    post generate_response_console_api_ai_assistant_path
+
+    assert_response 400
+
+    post generate_response_console_api_ai_assistant_path, params: {
+      prompt: "foo",
+      record_id: record_without_assistant.id,
+      record_klass: record_without_assistant.class
+    }
+
+    assert_response 500
+  end
```

Comment:
co valid? jeden test mi prijde malo

Reply:
Pridam trochu lepsi testy, ale API asi neotestuju, leda s dummy objektem, protoze to vyzaduje api key a kazdy request stoji penize. Zkusim aspon jestli existuji endpointy.

Reply:
A neslo by [VCR](https://github.com/vcr/vcr)? Klic by se pak v tom vygenerovanem ymlu mohl nahradit tim z env sample